Tests/OneDimensionalArraysTests.cs

```diff
@@ -0,0 +1,281 @@
+            }
+            catch (ArgumentNullException ex)
+            {
+                Assert.AreEqual("Array is empty", ex.ParamName);
+                Assert.Pass();
+            }
+
+            Assert.Fail();
+        }
+
+        [TestCase(new int[] { 5, 54, -9, 1, 0, -19, 6 }, 5)]
+        [TestCase(new int[] { 1 }, 0)]
+
+        public void GetMinIndexInArray_WhenArrIsNotNull_ShouldReturnMinNumInArr(int[] arr, int expeectedResult)
+        {
+            int actualResult = ProjLibrary.OneDimensionalArrays.MinIndexInArray_3(arr);
+            Assert.AreEqual(expeectedResult, actualResult);
+        }
+
+        [TestCase(null)]
+        [TestCase(new int[] { })]
+
+        public void GetMinIndexInArray_WhenArrIsNull_ShouldThrowArgumentNullException(int[] arr)
+        {
+            try
+            {
+                ProjLibrary.OneDimensionalArrays.MinIndexInArray_3(arr);
+            }
+            catch (ArgumentNullException ex)
+            {
+                Assert.AreEqual("Array is empty", ex.ParamName);
+                Assert.Pass();
+            }
+
+            Assert.Fail();
+        }
+
+        [TestCase(new int[] { 5, 54, -9, 1, 0, -19, 6 }, 1)]
+        [TestCase(new int[] { 1 }, 0)]
+
+        public void GetMaxIndexInArray_WhenArrIsNotNull_ShouldReturnMinNumInArr(int[] arr, int expeectedResult)
+        {
+            int actualResult = ProjLibrary.OneDimensionalArrays.MaxIndexInArray_4(arr);
+            Assert.AreEqual(expeectedResult, actualResult);
+        }
+
+        [TestCase(null)]
+        [TestCase(new int[] { })]
+
+        public void GetMaxIndexInArray_WhenArrIsNull_ShouldThrowArgumentNullException(int[] arr)
+        {
+            try
+            {
+                ProjLibrary.OneDimensionalArrays.MaxIndexInArray_4(arr);
+            }
+            catch (ArgumentNullException ex)
+            {
+                Assert.AreEqual("Array is empty", ex.ParamName);
+                Assert.Pass();
+            }
+
+            Assert.Fail();
+        }
+
+        [TestCase(new int[] { 5, 54, -9, 1, 0, -19, 6 }, 36)]
+        [TestCase(new int[] { 0, 1, 2, 3, 4, 5, 6 }, 9)]
+        [TestCase(new int[] { 1 }, 0)]
+
+        public void SummOfItemsWithOddIndexesInArray_WhenArrIsNotNull_ShouldCalculate(int[] arr, int expeectedResult)
+        {
+            int actualResult = ProjLibrary.OneDimensionalArrays.SummOfItemsWithOddIndexesInArray_5(arr);
+            Assert.AreEqual(expeectedResult, actualResult);
+        }
+
+        [TestCase(null)]
+        [TestCase(new int[] { })]
+
+        public void SummOfItemsWithOddIndexesInArray_WhenArrIsNull_ShouldThrowArgumentNullException(int[] arr)
+        {
+            try
+            {
+                ProjLibrary.OneDimensionalArrays.SummOfItemsWithOddIndexesInArray_5(arr);
+            }
+            catch (ArgumentNullException ex)
+            {
+                Assert.AreEqual("Array is empty", ex.ParamName);
```

Comment:
Param name should be like field name or variable name. `array`, `index` etc. But not  `Array is empty`. It's not a param.